### PR TITLE
some general fixes/additions for next release

### PR DESCRIPTION
**Pull request by antgonza**

### CODE REVIEW

File: qiita_db/test/test_processing_job.py [hunk 1]
```diff
@@ -457,13 +461,18 @@ def test_complete_success(self):
                                             'artifact_type': 'BIOM'}}
         job = _create_job()
         job._set_status('running')
+
+        # here we can test that job.release_validator_job hasn't been created
+        # yet so it has to be None
+        self.assertIsNone(job.release_validator_job)
```
Comment (ElDeveloper) [inline qiita_db/test/test_processing_job.py:467]: Can you add another test for the case where the return value is not None?

Comment (antgonza) [reply qiita_db/test/test_processing_job.py:467]: Ha! I knew I was forgetting something ... thank you!

**Comment by ElDeveloper** [comment]
Great, thanks @antgonza!
